Dockerfile

```diff
@@ -0,0 +1,15 @@
+#
+# This Dockerfile builds node-disk-manager
+# 
+
+#Set the base image
+FROM ubuntu:16.04
```

Comment:
Can you make this to be configurable like:
- Use a custom OS image with some pre-installed libraries:
  Refer: https://github.com/kubernetes/node-problem-detector/blob/master/Dockerfile.in

- Make the ARCH (amd64) configurable:
  Refer: https://github.com/kubernetes-incubator/spartakus/blob/master/Dockerfile

Reply:
sure @kmova